### PR TITLE
migrate to github actions

### PR DESCRIPTION
**Pull request by d-telnyx**

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    env:
+      TELNYX_MOCK_VERSION: 0.8.10
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```
Comment (lucasassisrosa) [inline .github/workflows/node.js.yml:20]: we still support node v8 on this. Can we add node 8 in here?

**Review by lucasassisrosa — COMMENTED** [review]
left a note on node versions

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (d-telnyx) [reply .github/workflows/node.js.yml]: Done, however I'd like to move to dropping support soon and **only** supporting the actively maintained versions (https://nodejs.org/en/about/releases/) with v10 end-of-life coming in spring 2021

Comment (lucasassisrosa) [reply .github/workflows/node.js.yml]: agreed.

**Review by lucasassisrosa — APPROVED** [review]

**Pull request merged**